### DISCUSSION TITLE
Plans to add Azure Blob Storage as part of the Supported File Location

### DISCUSSION THREAD

**Discussion by luanmorenomaciel** [astronomer/astro-sdk#1123] [Q&A]
Hi team,

Just wondering if the team has plans to add support for Azure Blob Storage in order to use the **load_file** operator that will unlock interesting capabilities for a multi-cloud ingestion platform

**Comment by pankajastro** [comment]
Hey @luanmorenomaciel  I think this is already in the roadmap you can track https://github.com/astronomer/astro-sdk/issues/905 issue
